.vscode/extensions.json

```diff
@@ -0,0 +1,3 @@
+{
+  "recommendations": ["rslfrkndmrky.rsl-vsc-focused-folder"]
```

Comment:
Should we suggest Prettier, since that's what we do on the CI? Or should we just leave it on the CI's job?

Reply:
We can probably add ESLint.
We could also add Prettier so that "format document" works as expected, but I don't think we should enable "lint on save".

Reply:
On second though, I think it would be more appropriate if we have `package.json` to bind specific versions of ESLint and Prettier.

> *ahem* one step to build step *ahem*